test/unit_tests/braket/tasks/test_local_quantum_task.py

```diff
@@ -29,33 +29,45 @@
     values=None,
 )
 
-TASK = LocalQuantumTask(RESULT)
-
 
 def test_id():
     # Task ID is valid UUID
+    TASK = LocalQuantumTask(RESULT)
+    TASK.result()
     uuid.UUID(TASK.id)
 
 
 def test_state():
-    assert TASK.state() == "COMPLETED"
+    TASK = LocalQuantumTask(RESULT)
+    assert TASK.state() == "CREATED"
 
 
 def test_result():
+    TASK = LocalQuantumTask(RESULT)
+    result = TASK.result()
+    assert result == RESULT
     assert RESULT.task_metadata.id == TASK.id
-    assert TASK.result() == RESULT
 
 
-@pytest.mark.xfail(raises=NotImplementedError)
 def test_cancel():
-    TASK.cancel()
+    num_qubits = 5
+    qc = Circuit()
+    for qubit in range(num_qubits):
+        qc.h(qubit)
+
+    for i in range(num_qubits - 1):
+        qc.cnot(i, i + 1)
+    task = LocalQuantumTask().create(qc, "default", 1000)
```

Comment:
Suggestion for better circuit so it doesn't finish before `task.cancel()` is invoked ?